website/www/site/content/en/documentation/io/developing-io-java.md

```diff
@@ -17,9 +17,6 @@ limitations under the License.
 -->
 # Developing I/O connectors for Java
 
-**IMPORTANT:** Use ``Splittable DoFn`` to develop your new I/O. For more details, read the
-[new I/O connector overview](/documentation/io/developing-io-overview/).
```

Comment:
There are still several changes like this that look like you've reverted changes that happened on master already - see also create-external-table.md, and dsls/dataframes/overview.md

Reply:
okay I will just take contents from website-revamp branch